priv/repo/modify_task_list_done.exs

```diff
@@ -0,0 +1,40 @@
+defmodule CodeCorps.Repo.Seeds.ModifyTaskListDone do
+  import Ecto.Query
+
+  alias CodeCorps.Project
+  alias CodeCorps.Repo
+  alias CodeCorps.Task
+  alias CodeCorps.TaskList
+
+  def migrate_existing() do
+    TaskList
+    |> Repo.all()
+    |> Enum.each(&update_done_column/1)
+
+    Project
+    |> preload(:task_lists)
+    |> Repo.all()
+    |> Enum.each(&update_tasks_to_done(&1))
+  end
+
+  defp update_tasks_to_done(project) do
+    Task
+    |> CodeCorps.Helpers.Query.project_filter(%{project_id: project.id})
+    |> Repo.all()
+    |> Enum.each(&assign_task_to_done(&1, Enum.at(project.task_lists, 3)))
+  end
+
+  defp assign_task_to_done(task, task_list_done) do
+    Task.changeset(task, %{task_list_id: task_list_done.id})
+    |> Repo.update()
+  end
+
+  defp update_done_column(%TaskList{name: "Done"} = task_list) do
+    TaskList.create_changeset(task_list, %{done: true})
+    |> Repo.update()
+  end
+  defp update_done_column(task_list), do: task_list
```

Comment:
This is good, but will also need a similar fn for `%TaskList{name: "In Progress"}` to make it the `pull_requests` list.